host/hackrf-tools/src/hackrf_transfer.c

```diff
@@ -638,7 +638,7 @@ static void usage()
 	printf("\t[-b baseband_filter_bw_hz] # Set baseband filter bandwidth in Hz.\n");
 	printf("\tPossible values: 1.75/2.5/3.5/5/5.5/6/7/8/9/10/12/14/15/20/24/28MHz, default <= 0.75 * sample_rate_hz.\n");
 	printf("\t[-C ppm] # Set Internal crystal clock error in ppm.\n");
-	printf("\t[-H hw_sync] # Synchronize RX/TX to external input signal.\n");
+	printf("\t[-H hw_sync] # Synchronize RX/TX to external trigger input.\n");
```

Comment:
`-H` no longer takes an argument now, so I think this usage line should read:

`[-H] # Synchronize RX/TX to external trigger input.`